### PR TITLE
new send onchain flow

### PR DESCRIPTION
**Pull request by limpbrains**
Here is new Send flow. It is still not fully implemented, a few things are still missing, like custom tx fee selection. But I think it is ready to be merged.

Right now I don't delete old screens and components and keeping them as a reference, I will do it when everything will be ready.

### CODE REVIEW

File: src/store/types/user.ts [hunk 1]
```diff
@@ -7,7 +7,9 @@ export type TViewController =
 	| 'receiveAssetPicker'
 	| 'coinSelection'
 	| 'feePicker'
-	| 'numberPad';
+	| 'SendNavigation'
```
Comment (coreyphillips) [inline src/store/types/user.ts]: 1. For consistency we may want to rename `SendNavigation` to `sendNavigation` unless you would prefer switching the other views to Pascal case.
2. `sendNavigation` will need to be included in the `defaultUserShape` `viewController` object [here](https://github.com/synonymdev/bitkit/blob/0fe5f5834c8f4d74a3e6a57daf4b59a55b6b5537/src/store/shapes/user.ts#L18) like so: `sendNavigation: { ...defaultViewController }`

File: src/store/types/user.ts [hunk 2]
```diff
@@ -7,7 +7,9 @@ export type TViewController =
...
+	| 'numberPad'
+	| 'test1';
```
Comment (coreyphillips) [inline src/store/types/user.ts]: `test1` can be removed.

File: src/store/types/user.ts (hunk 1 above)
Comment (limpbrains) [reply src/store/types/user.ts]: fixed

File: src/store/types/user.ts (hunk 2 above)
Comment (limpbrains) [reply src/store/types/user.ts]: fixed

**Comment by coreyphillips** [comment]
The SendNavigation component requires a name change [here](https://github.com/limpbrains/bitkit/blob/8cd0ea518cf5dae02bda745e5d8898a332cb224d/src/navigation/bottom-sheet/SendNavigation.tsx#L32) as well.

**Review by coreyphillips — APPROVED** [review]

**Review by Jasonvdb — APPROVED** [review]

**Pull request merged**